### PR TITLE
Add PPP

### PR DESCRIPTION
**Pull request by zognin**
Part of #207.
Can be viewed [here](https://zognin.github.io/tp/team/zognin.html)

### CODE REVIEW

File: docs/team/zognin.md [hunk 1]
```diff
@@ -3,34 +3,60 @@ layout: page
...
-    * Developer Guide:
+  * User Guide:
+    * Added some recipe features. (Pull Request [#156](https://github.com/AY2122S1-CS2103T-W16-2/tp/pull/156)).
+    * Updated feature overview. (Pull Request [#233](https://github.com/AY2122S1-CS2103T-W16-2/tp/pull/233)).
+  * Developer Guide:
+    * Added user stories. (Pull Request [#78](https://github.com/AY2122S1-CS2103T-W16-2/tp/pull/78)).
+    * Added implementation details for the switching of tabs and editing of ingredients in orders/ recipes. (Pull Request [#158](https://github.com/AY2122S1-CS2103T-W16-2/tp/pull/158)).
+    * Added some manual testing instructions. (Pull Request [#239](https://github.com/AY2122S1-CS2103T-W16-2/tp/pull/239)).
 
 * **Community**:
```
Comment (EuniceSim142) [inline docs/team/zognin.md:59]: Do you think bugs found during ped can be counted here?

something like this?
Screenshot 2021-11-02 at 2 29 07 PM

File: docs/team/zognin.md [hunk 2]
```diff
@@ -3,34 +3,60 @@ layout: page
...
 
-* **New Feature**:
-    * What it does:
-    * Justification:
-    * Highlights:
-    * Credits:
+* **New Feature**: Implemented commands to add and delete ingredients of an order. (Pull request [#92](https://github.com/AY2122S1-CS2103T-W16-2/tp/pull/92)).
+  * What it does: Allows the user to add and delete ingredients of an order.
+  * Justification: Each order has a list of ingredients.
+    Users may want to change one ingredient in the list without affecting other ingredients.
```
Comment (EuniceSim142) [inline docs/team/zognin.md:18]: I think this sounds better
```suggestion
    Users may want to change one ingredient in the list without having to enter in all the other ingredients again.
```

File: docs/team/zognin.md [hunk 3]
```diff
@@ -3,34 +3,60 @@ layout: page
...
-Given below are my contributions to the project.
+### Summary of Contributions
 
-* **New Feature**:
-    * What it does:
-    * Justification:
-    * Highlights:
-    * Credits:
+* **New Feature**: Implemented commands to add and delete ingredients of an order. (Pull request [#92](https://github.com/AY2122S1-CS2103T-W16-2/tp/pull/92)).
+  * What it does: Allows the user to add and delete ingredients of an order.
```
Comment (EuniceSim142) [inline docs/team/zognin.md:16]: Would it look nicer to bold these?

Screenshot 2021-11-02 at 2 32 10 PM

**Review by EuniceSim142 — COMMENTED** [review]
LGTM, just some suggestions.

File: docs/team/zognin.md (hunk 1 above)
Comment (zognin) [reply docs/team/zognin.md:59]: I think can, but I don't want to include it for my PPP

File: docs/team/zognin.md (hunk 3 above)
Comment (zognin) [reply docs/team/zognin.md:16]: Yup

**Review by EuniceSim142 — APPROVED** [review]
LGTM!

**Pull request merged**